src/app/models/timeslot.rb

```diff
@@ -25,7 +25,7 @@ def initialize(start, stop)
     end
 
     def to_s(with_day: false)
-      "#{start_day + ' ' if with_day}#{start_time} - #{end_time}"
+      "#{start_day + ' ' if with_day}#{start_time} – #{end_time}"
```

Comment:
@pcantrell 1.9 and later has the `\uNNNN` escape. It needs curly braces `\u{NNNNN}` when more than 4 digits.

![image](https://cloud.githubusercontent.com/assets/46175/24312078/e8a2d8e4-10a4-11e7-9e13-7f361ca98272.png)